src/serverenvironment.cpp

```diff
@@ -2173,3 +2195,103 @@ void ServerEnvironment::deactivateFarObjects(bool _force_delete)
 		m_active_objects.erase(*i);
 	}
 }
+
+void ServerEnvironment::openPlayerDatabase(const std::string &savedir)
+{
+	// Determine which database backend to use
+	std::string conf_path = savedir + DIR_DELIM + "world.mt";
+	Settings conf;
+	bool succeeded = conf.readConfigFile(conf_path.c_str());
+	if (!succeeded || !conf.exists("player_backend")) {
+		// fall back to files
+		conf.set("player_backend", "files");
+		warningstream << "/!\\ You are using old player file backend. This backend is deprecated "
+				<< " and will be removed in next release /!\\" << std::endl
+				<< "You should switch soon to SQLite3 or PostgreSQL." << std::endl
+				<< "For SQLite3, add 'player_backend = sqlite3' to your world.mt." << std::endl
+				<< "For PostgreSQL, add 'player_backend = postgresql' and "
+				<< "'pgsql_player_connection = host=<pghost> user=<pguser> password=<pgpassword> "
+				<< "dbname=<pgdb>' to your world.mt." << std::endl
+				<< "After configuring new player_backend setting an automatic migration will be "
+				<< "executed by " << PROJECT_NAME << " on startup. Your old player files will be "
+				<< "moved to players.old folder inside world folder." << std::endl;
```

Comment:
@sfan5 it's for developers not end users, for a such thing i can add it to wiki.minetest.net